app/src/pages/inside/projectSettingsPageContainer/content/patternAnalysis/patternAnalysisContent/patternAnalysisContent.jsx

```diff
@@ -36,6 +36,7 @@ import PencilIcon from 'common/img/newIcons/pencil-inline.svg';
 import BinIcon from 'common/img/newIcons/bin-inline.svg';
 import CopyIcon from 'common/img/newIcons/copy-inline.svg';
 import { docsReferences, createExternalLink } from 'common/utils';
+import { PROJECT_SETTINGS_PATTERN_ANALYSIS_EVENTS } from 'components/main/analytics/events/ga4Events/projectSettingsPageEvents';
```

Comment:
We can use shorter import syntax

we use the alias for this path. You can find it in base webpack config

Reply:
Done.